pkgs/development/python-modules/configshell/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub, python }:
```

Comment:
Please have a look at the other expressions in this folder. The exact dependencies need to be passed.